llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/Utils/WinogradConstants.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_DIALECTS_DIALECT_LINALGEXT_UTILS_WINOGRAD_CONSTANTS_H_
+#define IREE_DIALECTS_DIALECT_LINALGEXT_UTILS_WINOGRAD_CONSTANTS_H_
+
+namespace mlir {
+namespace iree_compiler {
+namespace IREE {
+namespace LinalgExt {
+namespace Winograd {
+
+// This file contains the Winograd constant matrices for different
+// output tile sizes and some Winograd utilities
+
+constexpr const char *getWinogradAttrName() { return "winograd.matmul"; }
```

Comment:
Do we need this? I only see the attribute being set, but not used anywhere.

Reply:
This was relevant for the SPIRV path but isn't anymore. Will remove.